omod/src/main/webapp/import/assessItem.jsp

```diff
@@ -126,57 +126,58 @@
 </p>
 
 <springform:form commandName="assessItemForm">
-	Assessing Item ${assessItemForm.index + 1} <input
+		<spring:message code="assessingItem" />

```

Comment:
The message code for this item should include "metadatasharing.nameofthevariable". I think you forgot the "metadatasharing part of it in some of the variables, please fix this.